templates/java/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-maven_templates='HelloWorld'
+maven_templates=$(ls -d */ | sed 's/\//\n/g')
```

Comment:
Nit: This change assumes that all the templates inside `Java` can be built with `maven`. This will have to be modified if things change and that is also fine.

Reply:
Correct the current one is assuming the same. This is just improving it so we don't have to add the names of the folders every time we add a new template

Reply:
On the other hand, wondering is we should build these with `gdk` commands as that's we expect.

Reply:
Mmmm definitely gdk commands I think.

Reply:
I will hold off on trying to do this. Right now the gdk does not allow you to specify the path to the artifact and for the OTF framework it is inside one of the submodules